mkt/site/storage_utils.py

```diff
@@ -120,7 +120,7 @@ def storage_is_remote():
 
 
 def walk_storage(path, topdown=True, onerror=None, followlinks=False,
-                 storage=private_storage):
+                 storage=None):
```

Comment:
Let's see how many tests fail when I force all code to explicitly pass storage classes.